tests/e2e/test_e2e.py

```diff
@@ -158,23 +167,31 @@ def regression_bounded(model, test_fraction=0.02):
     [
         # LightGBM
         regression(lightgbm.LGBMRegressor(**LIGHTGBM_PARAMS)),
+        regression_with_missing_values(
+            lightgbm.LGBMRegressor(**LIGHTGBM_PARAMS)),
         classification(lightgbm.LGBMClassifier(**LIGHTGBM_PARAMS)),
         classification_binary(lightgbm.LGBMClassifier(**LIGHTGBM_PARAMS)),
 
         # LightGBM (DART)
         regression(lightgbm.LGBMRegressor(**LIGHTGBM_PARAMS_DART)),
+        regression_with_missing_values(
+            lightgbm.LGBMRegressor(**LIGHTGBM_PARAMS_DART)),
         classification(lightgbm.LGBMClassifier(**LIGHTGBM_PARAMS_DART)),
         classification_binary(lightgbm.LGBMClassifier(
             **LIGHTGBM_PARAMS_DART)),
 
         # LightGBM (GOSS)
         regression(lightgbm.LGBMRegressor(**LIGHTGBM_PARAMS_GOSS)),
+        regression_with_missing_values(
+            lightgbm.LGBMRegressor(**LIGHTGBM_PARAMS_GOSS)),
         classification(lightgbm.LGBMClassifier(**LIGHTGBM_PARAMS_GOSS)),
         classification_binary(lightgbm.LGBMClassifier(
             **LIGHTGBM_PARAMS_GOSS)),
 
         # LightGBM (RF)
         regression(lightgbm.LGBMRegressor(**LIGHTGBM_PARAMS_RF)),
+        regression_with_missing_values(
+            lightgbm.LGBMRegressor(**LIGHTGBM_PARAMS_RF)),
```

Comment:
Do we really need to test each LightGBM's boosting type (DART, GOSS, RF) with missing values? If I'm not mistaken, missing values are handled in the same way by the same code in all boosting types. So, I believe it will be enough to test only one algorithm. 

I think it will be better to add classification case instead.

Also, please add a separate group for regression with missing values to not break the logical grouping of `regression, classification, classification binary` which is used across the whole file for all supported libraries.

Reply:
I've added tests for each boosting type just for the sake of grouping, it's most likely not necessary. Agree, it would be better to make a separate group for such tests.